google-cloud-tools-plugin/resources/META-INF/plugin.xml

```diff
@@ -105,7 +105,10 @@
 
   <application-components>
     <component>
-      <implementation-class>com.google.cloud.tools.intellij.startup.CloudToolsPluginInitializationComponent</implementation-class>
+      <implementation-class>com.google.cloud.tools.intellij.appengine.AppEngineInitializationComponent</implementation-class>
```

Comment:
If we modularize it per @patflynn suggestion, I'd expect something like this:

-AppEngineInitializationComponent - initalizes all the app engine stuff (like was previously in CloudToolsPluginInitialization component
- ConflictingAppEnginePluginCheck - an application component that performs the conflict check